script.js

```diff
@@ -4,18 +4,45 @@ const numColors = 8;
 const numTiming = 5;
 const numTops = 3;
 const earningRate = 3605;
-const numberFormatter = new Intl.NumberFormat('en', { style: 'currency', currency: 'USD', maximumFractionDigits: 0, minimumFractionDigits: 0 });
+const currencyFormatter = new Intl.NumberFormat('en', { style: 'currency', currency: 'USD', maximumFractionDigits: 0, minimumFractionDigits: 0 });
+const numberFormatter = new Intl.NumberFormat();
 let amount = 0;
 
 function createBubble(cost, text) {
   const left = Math.floor(Math.random() * 65);
   const aside = document.createElement("aside");
   aside.style.left = `${left + 5}vw`;
   aside.className = `color${Math.floor(Math.random() * numColors)} time${Math.floor(Math.random() * numTiming)} top${Math.floor(Math.random() * numTops)}`;
-  aside.innerHTML = `<b>${Math.floor(cost / earningRate)} seconds</b>:<br>${text}<br/><i>${numberFormatter.format(cost)}</i>`;
+  aside.innerHTML = `<b>${prettyPrintTime(Math.floor(cost / earningRate))}</b>:<br>${text}<br/><i>${currencyFormatter.format(cost)}</i>`;
   body.append(aside);
 }
 
+function prettyPrintTime(sec_num) {
+  const days = Math.floor(sec_num / 60 / 60 / 24); // minutes / hours / days
+  const hours = Math.floor((sec_num - (days * 24 * 60 * 60)) / 60 / 60); // subtract # days (in seconds) / minutes / hours
+  const minutes = Math.floor((sec_num - (days * 24 * 60 * 60) - (hours * 60 * 60)) / 60); // subtract # days and hours / minutes
+  const seconds = sec_num - (days * 60 * 60 * 24) - (hours * 60 * 60) - (minutes * 60) // subtract # days, hours, and minutes for remaining seconds
```

Comment:
Dumb question, could this math be simplified/cleaned up by using mod (%)? Like so?

```suggestion
  const hours = Math.floor(sec_num % days / 60 / 60); // subtract # days (in seconds) / minutes / hours
  const minutes = Math.floor(sec_num % days % hours / 60); // subtract # days and hours / minutes
  const seconds = sec_num % days % hours % minutes // subtract # days, hours, and minutes for remaining seconds
```

Reply:
Nice - that does definitely look cleaner, although it requires a little more brain power to read/understand the code. I'm not sure @alvaromontoro plans on merging in this code, anyway, though.

Reply:
In any case, this wouldn't have any performance impact (unless modulus is cheaper than multiplications + subtractions -- I haven't a clue), though with this change the comments could do with going away as well without loss of clarity. I do think it'd be slightly easier to debug for future maintainers. 

But, definitely agreed. At the end of the day this is bike shedding. The repo owner's schedule is definitely the limiting factor here.

Reply:
Definitely don't do what I said. If days or hours or minutes are 0, it'll throw a division by 0 error.